charts/portal/templates/deployment-backend-services.yaml

```diff
@@ -149,17 +149,17 @@ spec:
         - name: "SERVICES__CATENAADMINROLES__0__USERROLENAMES__0"
           value: "{{ .Values.backend.services.catenaAdminRoles.role0 }}"
         - name: "SERVICES__UPLOADSERVICEDOCUMENTTYPEIDS__0__DOCUMENTTYPEID"
-          value: "{{ .Values.backend.appmarketplace.type0 }}"
+          value: "{{ .Values.backend.appmarketplace.uploadAppDocumentTypeIds.documentTypeId0 }}"
```

Comment:
Please refer to services.uploadServiceDocumentTypeIds for the services service
https://github.com/eclipse-tractusx/portal-cd/blob/8f1b6f67e76c38f9d96e2b6932d12f50a55660e3/charts/portal/values.yaml#L594

Reply:
fixed